docs/specification/overview.md

```diff
@@ -0,0 +1,37 @@
+# Specifications
+
+ValueFlows aims at providing model which can get used in different information ecosystems.
+
+## The Web of Linked Data
+
+We provide web ontology which uses [RDF Schema 1.1](https://www.w3.org/TR/rdf-schema/) and [OWL 2](https://www.w3.org/TR/owl2-overview/) constructs. It builds on top of well established standards:
+
+* [IRIs](https://tools.ietf.org/html/rfc3987) as global identifiers
+* [HTTP/2](https://tools.ietf.org/html/rfc7540) as universal dereferencing protocol
+* [RDF 1.1](https://www.w3.org/TR/rdf11-concepts/) as well defined (web native) data graph model
+* [SHACL](https://www.w3.org/TR/shacl/) and [ShEx](https://shex.io/) as constraints and validation languages
+* [Web Architecture](https://www.w3.org/standards/webarch/) as omnipresent platform
+
+This leads to certain advantages:
+
+* namespacing helps prevent naming collisions
+* abstract data model which can adopt new serializations, eg. [HDT](http://www.rdfhdt.org/)
```

Comment:
TODO: mention CBOR https://forum.solidproject.org/t/hdt-for-dynamic-and-small-graphs/163/4